CHANGELOG.md

```diff
@@ -14,7 +14,9 @@ Changelog for Cass Operator, new PRs should update the `main / unreleased` secti
 * [CHANGE] #213 Integration tests in Github Actions are now reusable across different workflows
 * [FEATURE] #193 Add new Management API endpoints to HTTP Helper: GetKeyspaceReplication, ListTables, CreateTable
 * [FEATURE] #175 Add FQL reconciliation via parseFQLFromConfig and SetFullQueryLogging called from ReconcileAllRacks. CallIsFullQueryLogEnabledEndpoint and CallSetFullQueryLog functions to httphelper.
+* [FEATURE] Allow overriding default Cassandra and DSE repositories and give versions a default suffix
```

Comment:
Need to add the PR number here

Reply:
That's not available when making the PR (these always need to be changed afterwards).